php-nginx/php.ini

```diff
@@ -478,7 +478,7 @@ error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT
 ; Development Value: On
 ; Production Value: Off
 ; http://php.net/display-errors
-display_errors = Off
+display_errors = stderr
```

Comment:
When I tested these configs, the logs sent to STDERR don't appear anywhere. Also, it is possible to override any `php.ini` settings by having `php.ini` file at the top level directory of your web app. I would not change the default values to something less strict.

Good news is that I think I found an alternate solution.

When I set `catch_workers_output = yes` in `/usr/local/php/etc/php-fpm.conf`, I see the logs in `/var/log/app_engine/php-fpm.log`. 

Does it work for you?